frontend/README.md

```diff
@@ -0,0 +1,24 @@
+# authentication
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
+
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
```

Comment:
Feel free to remove this file and keep some similar instruction at the root folder

Reply:
Yes, I have prepared a documentation. I will put that here.